Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -127,6 +127,40 @@ namespace
         }
     }
 
+    constexpr std::array<const char*, 25> HAND_JOINT_NAMES
+    {
+        "WRIST",
+
+        "THUMB_METACARPAL",
```

Comment:
should these values be all capital with underscores now that they are exposed as XRHand variables? I have typically seen WebXR string definitions as all lower case with dashes.

Reply:
nevermind, it looks like this is correct relative of the spec: [https://immersive-web.github.io/webxr-hand-input/](https://immersive-web.github.io/webxr-hand-input/)

---
In reply to: [496088316](https://github.com/BabylonJS/BabylonNative/pull/421#discussion_r496088316) [](ancestors = 496088316)

Reply:
I'm naming them to match https://immersive-web.github.io/webxr-hand-input/#xrhand-interface